lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -99,7 +99,27 @@ class ReportUIFeatures {
     window.addEventListener('resize', this._updateStickyHeaderOnScroll);
     this._setupMetricDescriptionToggleElements();
     const topbarLogo = this._dom.find('.lh-topbar__logo', this._document);
-    topbarLogo.addEventListener('click', this._toggleDarkTheme);
+    topbarLogo.addEventListener('click', () => this._toggleDarkTheme());
+
+    let turnOffTheLights = false;
+    if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
```

Comment:
sweeet, I guess unrecognized params just default to `.matches` false in other browsers?

Reply:
yup